builtin/game/chatcommands.lua

```diff
@@ -559,6 +559,14 @@ local function handle_give_command(cmd, giver, receiver, stackstring)
 		return false, "Cannot give an empty item"
 	elseif not itemstack:is_known() then
 		return false, "Cannot give an unknown item"
+	--[[ We forbid giving 'ignore' because it's useless as an item and
+	also a bit dangerous.
+	Note this does NOT make it completely impossible for players to
+	get 'ignore'; they might still obtain it via mods or bugs in mods,
+	so beware. ]]
```

Comment:
KISS:
`-- Forbid obtaining ignore due to unwanted side effects`

Comment below does not add anything meaningful to the code, please remove it.

Reply:
Huh? The fact that this check does not perfeclty prevent obtaining `ignore` IS absolutely significant, the discussion above shows that paramat did not consider this and would have been tempted to remove neccessary code.

Also: Are you complaining about me making too many code comments? OMG.

```
* Wuzzy dies a little bit inside.
```

Reply:
It is obvious that a change in the `/give` command does not affect other mods which can give any item to the player inventory. If there is a place to document the consequences of placing ignore, then it's the wiki page and/or in a Minetest manual. Comments exist to provide additional information which is relevant in this context. For example `so be aware` sounds like a guide entry for "How to make a Minetest mod" - having it here as comment in the code does not help anyone.

Reply:
I repeat again: Paramat (!!!) did not see this initially, so if even core devs can be fooled, I think it is NOT obvious to everyone and a comment is justified. Note I have shortened the comment a bit now, I don't know why it is not visible here.
I like to hear an opinion of another core dev here.

Reply:
I agree with SmallJoker.